fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java

```diff
@@ -260,6 +260,8 @@ private void matchGroup() {
                                     } else {
                                         tablet.setLastStatusCheckTime(checkStartTime);
                                     }
+                                } else {
```

Comment:
How to add a test case to avoid the problem occurring again?

Reply:
I will add some test cases when doing the colocate optimization. But this bugfix should be merged into 2.4. So we can merge it before the test cases are added.